### ISSUE TITLE
How to deal with CSS4 font-weight allowing font-weight 1...999?

### ISSUE THREAD

**Issue by davelab6** [typekit/fvd#6]
How to deal with CSS4 font-weight allowing font-weight 1...999?

Over in https://github.com/google/fonts/issues/20 a Google Fonts user suggested that Google Fonts adopt this, but with CSS4 in mind, it seems fvd wouldn't work well in the CSS4 future.

**Comment by bramstein** [comment]
@davelab6 Yes, we're slowly moving away from using FVD's at Typekit. The replacement is just straight up font-weight, font-style, and font-stretch.

**Comment by davelab6** [comment]
Okay, cool, thanks for confirming :)

**Comment by quitequinn** [comment]
I'm wondering what we should be moving towards. This convention was great for CSS3 but now that we are building for CSS4 it would be helpful to have a convention to lean on. Is the industry relying on name table norms for style selection or is there a collaboratively developed convention that I'm unaware of that deals with optical sizes, 9+ weights, etc.

**Comment by quitequinn** [comment]
Basically, has a better approach coalesced in the last 4 years than... 
font-style=normal&font-weight=425&font-width=511&font-stretch=221

**Comment by davelab6** [comment]
Width is stretch in CSS. GF uses axis tag codes
